Rlog/Model/Weekday.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Weekday.swift
+//  Rlog
+//
+//  Created by Noah's Ark on 2022/11/12.
+//
+
+import Foundation
+
+enum Weekday: String, CaseIterable {
+    case mon = "MON"
+    case tue = "TUE"
+    case wed = "WED"
+    case thu = "THU"
+    case fri = "FRI"
+    case sat = "SAT"
+    case sun = "SUN"
```

Comment:
```suggestion
    case sun = "SUN"
    case mon = "MON"
    case tue = "TUE"
    case wed = "WED"
    case thu = "THU"
    case fri = "FRI"
    case sat = "SAT"
```
마지막에 언급한 요일 인덱싱 관련 이슈와
요일이 실재와 다르게 표시되는 이슈는 이렇게 수정하면 해결될것같습니다. CaseIterable이 작성 순서대로 리스트화해주는 이유때문인것같습니다.

PR본문중 스크린샷과 제가 빌드한 모습(하단 스크린샷)이 달라 보이는것을 생각해보면 코드가 옮겨지면서 누락된듯합니다!!

![image](https://user-images.githubusercontent.com/46256034/201534386-87092cab-ffd2-4041-85f9-73dc8f21ab16.png)

Reply:
[c7455b8](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/72/commits/c7455b89c011237f46e254a74b0042440198ea8a) 반영했습니다.